student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(name: 'Unknown', age: 0, parent_permission: true, classroom: 'Unknown')
```

Comment:
- [ ] [OPTIONAL] great job setting up the `Student` class 👏 it is not actually necessary for `@age` or `@classroom` to have a default value in this case.